RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,79 @@
+# Bitcoin Definitions
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 26 Feb. 2019
+
+# Table of contents
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+- [Registry extension](#registry-extension)
+
+## Description
+
+Introduces definitions to allow the execution of RFC-003 for Bitcoin asset on the [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+TODO: Decide we if do 1 RFC that introduce general Bitcoin terms + RFC003 or 2 RFCs (one for each)
```

Comment:
2 RFCs:
- do one thing per RFC
- if superseding needed for BTC RFC-003 then no issue
- RFC-003 for bitcoin is mostly bitcoin script related (hex, etc). It makes to be on its own
- Allow better separation of blockchains (litecoin, bitcoin cash) making it easier if there are technical differences.

Decision: 2 RFCs.
1. General terms
2. RFC-003 specific